packages/contracts-bridge/contracts/test/BridgeToken.t.sol

```diff
@@ -0,0 +1,687 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+pragma solidity 0.7.6;
+
+import {MockWeth} from "./utils/MockWeth.sol";
+import "forge-std/Test.sol";
+
+// Tests are largely based on ERC20Test from solmate
```

Comment:
we can't include code directly from solmate due to license restrictions. Can you link the specific files that you referenced so we can check the overlap?

Reply:
Most of the tests that test the ERC20 functions that are inherited and used by BridgeToken.

Reply:
Can you link the specific files that you referenced so we can check the overlap?

Reply:
is comment now redundant?